packages/react-instantsearch/examples/react-router/src/App.js

```diff
@@ -0,0 +1,99 @@
+import React, {Component} from 'react';
+import {
+    InstantSearch, HierarchicalMenu,
+    Hits, Menu, Pagination, PoweredBy, RangeRatings,
+    RefinementList, SearchBox, ClearAll,
+} from 'react-instantsearch/dom';
+/* eslint-disable import/no-unresolved */
```

Comment:
Yes I added that because it was failing on travis saying that he couldn't find react-router module (but everything went perfectly fine locally).